src/ProjectTemplates/test/MvcTemplateTest.cs

```diff
@@ -107,7 +107,7 @@ private async Task MvcTemplateCore(string languageOverride)
         [ConditionalTheory]
         [InlineData(true)]
         [InlineData(false)]
-        [SkipOnHelix("ef restore no worky")]
+        [SkipOnHelix("cert failure", Queues = "OSX.1014.Amd64;OSX.1014.Amd64.Open")]
```

Comment:
Would this problem go away if we were testing against the final SDK i.e. w/ notarized packages for Catalina?

Reply:
No idea, I'm not familiar with what the cert requirements are, I don't think we had this issue on the azdo machines tho, so probably something specific to the helix macs